### PR TITLE
Version method for kernel_module

### PR DESCRIPTION
**Pull request by aladmit**
Hi! This version method for kernel_module.

issue: #993

### CODE REVIEW

File: lib/resources/kernel_module.rb [hunk 1]
```diff
@@ -36,6 +36,16 @@ def loaded?
       !found.nil?
     end
 
+    def version
+      # default modinfo command
+      modinfo_cmd = "modinfo -F version #{@module}"
+      # command for CentOS 5 and sudo
+      modinfo_cmd = "/sbin/modinfo -F version #{@module}" if inspec.os[:name] == 'centos' && inspec.os[:release].to_i == 5
```
Comment (chris-rock) [inline lib/resources/kernel_module.rb]: maybe:

```
if inspec.os.name == 'centos' && inspec.os.release.to_i == 5
   ...
else
   modinfo_cmd = "modinfo -F version #{@module}"
end

```

File: lib/resources/kernel_module.rb [hunk 2]
```diff
@@ -36,6 +36,16 @@ def loaded?
...
+
+      cmd = inspec.command(modinfo_cmd)
+      cmd.exit_status.zero? ? cmd.stdout.gsub("\n", '') : false
```
Comment (chris-rock) [inline lib/resources/kernel_module.rb]: It may be better to return `nil`, since we could not determine a version

**Review by chris-rock — COMMENTED** [review]
Thank you @postgred for this great addition. I added ideas for minor improvements.

Right now, rubocop is failing with:
```
Offenses:
lib/resources/kernel_module.rb:46:42: C: Use delete instead of gsub.
      cmd.exit_status.zero? ? cmd.stdout.gsub("\n", '') : false
                                         ^^^^^^^^^^^^^^
```

Could you please sign-off your work, too. Instructions are available here: https://github.com/chef/inspec/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco

**Comment by chris-rock** [comment]
@postgred Thank you for the quick improvements. The implementation looks great. I was just thinking, we should add this to our integration tests as well, to ensure it works across linux systems. The test is located here: https://github.com/chef/inspec/blob/master/test/integration/default/kernel_module_spec.rb#L21-L23

This will be executed with every travis run. See https://github.com/chef/inspec/blob/master/.travis.yml#L28-L55

**Comment by aladmit** [comment]
@chris-rock We can't add this to integration tests because **modinfo** can't work in docker. Example:

```
root@580f55841391:/# modinfo video
libkmod: ERROR ../libkmod/libkmod.c:556 kmod_search_moddep: could not open moddep file '/lib/modules/4.9.5-200.fc25.x86_64/modules.dep.bin'
modinfo: ERROR: Module alias video not found.
```

**Comment by chris-rock** [comment]
@postgred Agreed. Thank you.

**Pull request merged**